src/templates/home.tsx

```diff
@@ -24,25 +24,13 @@ export default Home
 // TODO: Use fragments for these page queries
 // and share them between the doc template and this homepage
 export const pageQuery = graphql`
-  query($repositoryName: String!, $repositoryOwner: String!) {
+  query {
     site {
       siteMetadata {
         title
         docsLocation
       }
     }
-    github {
```

Comment:
This was removed temporarily to allow it to run locally without needing to connect to GitHub. The parameters `$repositoryName: String!, $repositoryOwner: String!` were removed as a result. It should all be reverted before completing this PR.